applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1366,7 +1367,8 @@ get_caller_id_number(Call) ->
     CallerIdNumber = whapps_call:caller_id_number(Call),
     case whapps_call:kvs_fetch('prepend_cid_number', Call) of
         'undefined' -> CallerIdNumber;
-        Prepend -> <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>
+        Prepend -> Pre = <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>,
+                   wh_util:truncate_right_binary(Pre, 35)
```

Comment:
How about these places:

applications/stepswitch/src/stepswitch_util.erl: Line 129
core/whistle-1.0.0/src/wh_json.erl: Line 123
applications/call_inspector/src/parsers/ci_parser_freeswitch.erl: Line 340
applications/call_inspector/src/parsers/ci_parsers_util.erl: Line 108

Using a meaningful function name helps code reading and makes coder’s intention clear.

From: Pierre Fenoll <notifications@github.com<mailto:notifications@github.com>>
Reply-To: 2600hz/kazoo <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, December 3, 2015 at 11:03 AM
To: 2600hz/kazoo <kazoo@noreply.github.com<mailto:kazoo@noreply.github.com>>
Cc: Hesaam Farhang <hesaam@2600hz.com<mailto:hesaam@2600hz.com>>
Subject: Re: [kazoo] KAZOO-4321: Truncate prepend caller id to not exceed characters long limit (#1465)


In applications/callflow/src/module/cf_voicemail.erl<https://github.com/2600hz/kazoo/pull/1465#discussion_r46594988>:

> @@ -1366,7 +1367,8 @@ get_caller_id_number(Call) ->
>      CallerIdNumber = whapps_call:caller_id_number(Call),
>      case whapps_call:kvs_fetch('prepend_cid_number', Call) of
>          'undefined' -> CallerIdNumber;
> -        Prepend -> <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>
> +        Prepend -> Pre = <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>,
> +                   wh_util:truncate_right_binary(Pre, 35)


Well, I'm happy with it if you also find more than one other place in master to use it, and call the function there too.

—
Reply to this email directly or view it on GitHub<https://github.com/2600hz/kazoo/pull/1465/files#r46594988>.